yang-module-versioning/draft-ietf-netmod-yang-module-versioning.txt

```diff
@@ -1028,7 +1040,7 @@ Internet-Draft    Updated YANG Module Revision Handling    February 2024
        "This YANG 1.1 module contains definitions and extensions to
         support updated YANG revision handling.
 
-        Copyright (c) 2002 IETF Trust and the persons identified as
+        Copyright (c) 2024 IETF Trust and the persons identified as
```

Comment:
I'll update the YANG files too.

Reply:
I didn't realise that we were still maintaining the YANG files separately (although that would be best with some better tooling), but I make some other changes to the YANG to fix the line length issues reported by xml2rfc.

Reply:
I extracted the YANG files out because yangvalidator was complaining with a previous rev and just from the full document I couldn't figure it out.